src/index.ts

```diff
@@ -571,7 +571,7 @@ export class J5IO extends AbstractIO {
   public i2cWrite(address: number, register: number): void;
   public i2cWrite(address: number, inBytes: number[]): void;
   public i2cWrite(address: number, register: number, inBytes: number[]): void;
-  public i2cWrite(address: number, registerOrInBytes: number | number[], inBytes?: number[]): void {
+  public i2cWrite(address: number, registerOrInBytes: number | number[], inBytes?: number | number[]): void {
```

Comment:
To confirm, is the new specific signature being called as shown below?

```typescript
i2cWrite(address: number, register: number, inBytes: number): void;
```

If so, we'll want to squeeze this line between line 572 and 573. In TypeScript, the last signature without a semicolon following it is a "merged" signature. It's not one that's actually used by calling TypeScript code, but it encompasses all of the signatures that end with a semicolon to present the "internal"/vanillaJS meta signature.